### DISCUSSION TITLE
Cordova-ios 6.2.0

### DISCUSSION THREAD

**Discussion by chrux** [timbru31/cordova-plugin-lottie-splashscreen#305] [Q&A] [answered]
Hey,

In the documentation I see a cordova-ios >= 5.0.0 only, was wondering, does this plugin support the latest version of cordova-ios 6.2.0?

Best

**Comment by timbru31** [comment, marked as answer]
Yes of course! That implies the `>` :)  
In fact you are always encourages to use the latest available cordova-ios version.
